### PR TITLE
Adapted tests to php 8 and symfony 5 components

### PR DESCRIPTION
**Pull request by chives**

### CODE REVIEW

File: Tests/DataGrid/Extension/Configuration/EventSubscriber/ConfigurationBuilderTest.php [hunk 1]
```diff
@@ -25,179 +25,172 @@
...
-                $bundle = $this->createMock(Bundle::class);
-                $bundle->expects($this->any())
-                    ->method('getPath')
-                    ->will($this->returnValue(sprintf('%s/../../../../Fixtures/FooBundle', __DIR__)))
-                ;
-
-                return [$bundle];
-            }));
+            ->willReturnCallback(
+                function () {
```
Comment (szymach) [inline Tests/DataGrid/Extension/Configuration/EventSubscriber/ConfigurationBuilderTest.php:57]: `:void`

File: Tests/DataGrid/Extension/Configuration/EventSubscriber/ConfigurationBuilderTest.php [hunk 2]
```diff
@@ -25,179 +25,172 @@
...
-                $barBundle = $this->createMock(Bundle::class);
-                $barBundle->expects($this->any())
-                    ->method('getPath')
-                    ->will($this->returnValue(sprintf('%s/../../../../Fixtures/BarBundle', __DIR__)))
-                ;
-
-                return [$fooBundle, $barBundle];
-            }));
+            ->willReturnCallback(
+                function () {
```
Comment (szymach) [inline Tests/DataGrid/Extension/Configuration/EventSubscriber/ConfigurationBuilderTest.php:86]: `:void`

File: Tests/DataGrid/Extension/Configuration/EventSubscriber/ConfigurationBuilderTest.php [hunk 3]
```diff
@@ -25,179 +25,172 @@
...
-                $bundle = $this->createMock(Bundle::class);
-                $bundle->expects($this->any())
-                    ->method('getPath')
-                    ->will($this->returnValue(sprintf('%s/../../../../Fixtures/FooBundle', __DIR__)))
-                ;
-
-                return [$bundle];
-            }));
+            ->willReturnCallback(
+                function () {
```
Comment (szymach) [inline Tests/DataGrid/Extension/Configuration/EventSubscriber/ConfigurationBuilderTest.php:149]: `: void`

**Comment by chives** [comment]
@fsi-open

**Pull request merged**